src/Symfony/Bridge/Doctrine/Tests/ArgumentResolver/EntityValueResolverTest.php

```diff
@@ -54,6 +54,10 @@ public function testResolveWithoutAttribute()
 
     public function testResolveWithoutManager()
     {
+        if (method_exists(ObjectManager::class, 'isUninitializedObject')) {
+            $this->markTestSkipped('This test is not applicable to doctrine/persistence 4');
+        }
```

Comment:
How does Persistence 4 behave in this situation? Should we have a test for that as well?

Reply:
@derrabus `TypeError` in the signature of the `AbstractManagerRegistry` when passing `null` as default manager name. And `InvalidArgumentException` if there is no manager with that name.

@greg0ire we cannot remove support for the case of having no entity manager. As the ManagerRegistry interface extends ConnectionRegistry in doctrine/persistence (which is a mistake to me, but won't be an easy fix) and we don't have any standalone implementation of ConnectionRegistry, DoctrineBundle will still create a manager registry even when your project enables DBAL and not ORM. And so this case exists (it won't be a case of having no manager registry at all)

Reply:
the proper fix is to fix the mock of the ManagerRegistry. When passing `null` as argument here, `getManagerForClass` should indeed return `null` but `getManager` should throw an `InvalidArgumentException` (which is already the behavior of doctrine/persistence 2.x and 3.x btw)

Reply:
Thanks for the pointers, I'll try that 👍